src/test/lit-html_test.ts

```diff
@@ -324,6 +315,14 @@ suite('lit-html', () => {
         assert.equal(stripExpressionDelimeters(container.innerHTML), '<div foo="bar"></div>');
       });
 
+      test('renders a case-sensitive attribute', () => {
+        const size = 100;
+        render(html`<svg viewBox="0 0 ${size} ${size}"></svg>`, container);
+        assert.equal(
+            stripExpressionDelimeters(container.innerHTML),
+            '<svg viewBox="0 0 100 100"></svg>');
+      });
```

Comment:
For Edge test to pass, apparently need to strip the extra `xmlns` attribute when asserting.  And they helpfully render the innerHTML as a self-closing tag, apparently.  Sigh.
```
expected '<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100" />' to equal '<svg viewBox="0 0 100 100"></svg>'
```
Maybe just change it to 
```js
assert.match(stripExpressionDelimeters(container.innerHTML), /viewBox="0 0 100 100"/);
```